tests/Backend/ChainedTest.php

```diff
@@ -19,7 +19,7 @@
 class ChainedTest extends TestCase
 {
 
-    public function test_constructor_getbackends_shouldMakeSureToHaveProperIndex()
+    public function constructor_getbackends_shouldMakeSureToHaveProperIndexTest()
```

Comment:
the `test` needs to be a prefix and not a postfix. You may need change that for this and all other test methods again. Otherwise those tests will be simply skipped. You can see that in the action results, as it complains about certain files not containing any tests

Reply:
Indeed, sorry for my nonsense. Reverted and pushed